vllm/transformers_utils/configs/speculators/base.py

```diff
@@ -1,14 +1,40 @@
 # SPDX-License-Identifier: Apache-2.0
 # SPDX-FileCopyrightText: Copyright contributors to the vLLM project
 import os
-from typing import Any, Union
+from typing import Any, Optional, Union
 
 from transformers import PretrainedConfig
 
 from vllm.transformers_utils.configs.speculators.algos import (
     SUPPORTED_SPECULATORS_TYPES)
 
-__all__ = ["SpeculatorsConfig"]
+__all__ = [
+    "SpeculatorsConfig", "SpeculatorsProcessingError",
+    "load_speculators_config_dict"
+]
+
+
+class SpeculatorsProcessingError(Exception):
+    """Exception raised when speculators model processing fails."""
+    pass
+
+
+def load_speculators_config_dict(model: str) -> Optional[dict[str, Any]]:
```

Comment:
We have several other functions that are very similar to this. Instead of adding this method, I'd look into reusing something like [get_config](https://github.com/neuralmagic/vllm/blob/65a93f1e57c97d6e6934622271e82df9bc7e47ff/vllm/transformers_utils/config.py#L313) for example